### ISSUE TITLE
MIDI learn may currupt connections: Arc has multiple tails

### ISSUE THREAD

**Issue by loki42** [drobilla/ingen#23]
I'm not sure of the exact steps to reproduce but it happens with MIDI learn and adding / removing connections (arcs). Arc are created with multiple tails that the ingen will save but won't restore. Error on the Ingen side is: 

Arc has multiple tails

and they do:

```
_:b1
        ingen:head <uberheim_filter3/in> ,
                <uberheim_filter5/in> ;
        ingen:tail <in_3> ,
                <in_4> ;
        midi:controllerNumber 75 ;
        a midi:Controller .
```

Ingen file that has the shows the issue is attached.

[Aa.ingen.zip](https://github.com/drobilla/ingen/files/6126733/Aa.ingen.zip)

**Comment by drobilla** [comment]
Ah, that makes sense now.  I couldn't imagine anywhere arcs like that came from, but MIDI bindings are probably the sketchiest thing in there, so that will probably narrow it down enough.  I'll check it out, thanks.

**Comment by drobilla** [comment]
Should be fixed by f9014009
